### PR TITLE
(#5621) using KeyMayExist instead of Get when visit rocksdb

### PR DESCRIPTION
**Pull request by wangbo**
## Proposed changes

using KeyMayExist instead of Get when visit rocksdb.
see #5621

### CODE REVIEW

File: be/src/olap/rowset/rowset_meta_manager.cpp [hunk 1]
```diff
@@ -38,11 +38,7 @@ bool RowsetMetaManager::check_rowset_meta(OlapMeta* meta, TabletUid tablet_uid,
                                           const RowsetId& rowset_id) {
     std::string key = ROWSET_PREFIX + tablet_uid.to_string() + "_" + rowset_id.to_string();
     std::string value;
```
Comment (acelyc111) [inline be/src/olap/rowset/rowset_meta_manager.cpp:40]: No need to get value?

Comment (wangbo) [reply be/src/olap/rowset/rowset_meta_manager.cpp:40]: For doris, no need to get a value kere.
But rocksdb's ```KeyMayExist``` need a value.

Comment (acelyc111) [reply be/src/olap/rowset/rowset_meta_manager.cpp:40]: Yes,you are right

**Review by acelyc111 — APPROVED** [review]

**Review by caiconghui — APPROVED** [review]
+1

**Pull request merged**